tee-worker/client-api/parachain-api/prepare-build/interfaces/sidechain/definitions.ts

```diff
@@ -86,7 +86,7 @@ export default {
                 UnexpectedMessage: "Null",
                 __Unused_WrongSignatureType: "Null",
                 VerifyWeb3SignatureFailed: "Null",
-                NoEligibleIdentity: "Null",
+                __Unused_NoEligibleIdentity: "Null",
```

Comment:
I suggest removing it completely.

Imagine we have a new entry later, we might erroneously append them like:
```
// in rust
#[codec(index = 11)]
NewEntry,

// in definitions.ts
__Unused_NoEligibleIdentity: "Null",
NewEntry: "Null",
```

Which is wrong, so it's very error-prone IMO.